ni_measurementlink_service/_datatypeinfo.py

```diff
@@ -7,28 +7,23 @@
 
 
 class DataTypeInfo(NamedTuple):
-    """Class that represents the information for each of the DataType enum values.
+    """Class that represents the information for each of the DataType enum values."""
 
-    Attributes
-    ----------
-        grpc_field_type: Field.Kind associated with the DataType
+    grpc_field_type: type_pb2.Field.Kind.ValueType
+    """Field.Kind associated with the DataType."""
 
-        repeated: Whether the DataType is a repeated field
+    repeated: bool
+    """Whether the DataType is a repeated field."""
 
-        type_specialization: Specific type when value_type
-        can have more than one use
+    type_specialization: TypeSpecialization = TypeSpecialization.NoType
+    """Specific type when value_type can have more than one use."""
 
-        message_type (str): This is the gRPC full name of the message type.
-        Required when 'grpc_field_type' is Kind.TypeMessage.
-        Ignored for any other 'type'.
+    message_type: str = ""
+    """The gRPC full name of the message type. Required when 'grpc_field_type' is Kind.TypeMessage.
 
+    Ignored for any other 'type'.
```

Comment:
'grpc_field_type'

Reply:
Updated and converted to a real cross-reference.

(Then I noticed this file is internal, but whatever, Sphinx still creates docs for it.)